docs-source/content/samples/simple/domains/model-in-image/prerequisites.md

```diff
@@ -53,8 +53,8 @@ weight: 1
     - Option 1: To create the ingresses, use the following YAML file to create a file called `/tmp/mii-sample/ingresses/myingresses.yaml` and then call `kubectl apply -f /tmp/mii-sample/ingresses/myingresses.yaml`:
 
        ```yaml
-       apiVersion: networking.k8s.io/v1beta1
-       kind: Ingress
+       apiVersion: traefik.containo.us/v1alpha1
```

Comment:
There are indentation errors in a few places in this yaml.

Reply:
Fixed the indention issue